src/hooks/useDisputes.js

```diff
@@ -61,25 +63,46 @@ export default function useDisputes() {
   }, [disputesPhases, disputes, disputesPhasesKey, error]) // eslint-disable-line react-hooks/exhaustive-deps
 }
 
+/**
+ * Hook for single dispute
+ * @param {String} disputeId Id of the dispute
+ * @returns {Array} First element the dispute, second is the fetching indicator,
+ * third might be the error from the graph or an ipfs error,
+ * third a boolean that indicates if the error is from the graph since we need to handle in a different way
+ * in the dispute detail, the timeline can not be displayed if is a graph error but can if the error is from ipfs
+ */
 export function useDispute(disputeId) {
   const courtConfig = useCourtConfig()
   const now = useNow() // TODO: use court clock
-  const { dispute, fetching } = useSingleDisputeSubscription(disputeId)
-  const disputeProcessed = useProcessedDispute(dispute, fetching)
+  const {
+    dispute,
+    fetching: graphFetching,
+    error: graphError,
+  } = useSingleDisputeSubscription(disputeId)
+
+  const graphErrorMessage = graphError ? graphError.message : ''
+  const disputeProcessed = useProcessedDispute(dispute, graphFetching)
+  const error = graphErrorMessage || disputeProcessed.error
 
   const disputePhase = getPhaseAndTransition(dispute, courtConfig, now)
   const disputePhaseKey = disputePhase
     ? convertToString(Object.values(disputePhase)[0])
     : ''
 
   return useMemo(() => {
-    return {
-      dispute: {
-        ...disputeProcessed,
-        ...disputePhase,
-      },
-    }
-  }, [disputeProcessed, disputePhaseKey]) // eslint-disable-line react-hooks/exhaustive-deps
+    const fetching = graphFetching || disputeProcessed.fetching
+    return [
+      dispute
+        ? {
+            ...disputeProcessed,
+            ...disputePhase,
+          }
+        : null,
+      fetching,
+      error,
+      !!graphErrorMessage,
+    ]
+  }, [disputeProcessed, disputePhaseKey, error, graphFetching]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
I think we may want to temporarily remove the `eslint-disable` rule and check if we may have more props to add here

Reply:
If you have the editor integration set up, this should be easy to do: just remove the comment, run prettier, and then add the comment back.